system-addon/test/functional/mochitest/browser_highlights_section.js

```diff
@@ -0,0 +1,65 @@
+"use strict";
+
+add_task(async function setup() {
+  registerCleanupFunction(async () => {
```

Comment:
Why do we need to have a task to register cleanup?

Reply:
It was the usage I saw in a different test https://searchfox.org/mozilla-central/source/browser/modules/test/browser/browser_UsageTelemetry_content.js#34 and documentation is not great.

Reply:
The use case there is probably different. I'll clean it up.